src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Incident intelligence user settings."
+metaDescription: "Use the incident intelligence settings to set your grace period and inactive issue time policies."
+---
+
+Incident intelligence general settings page lets you set how much time our system spends observing new activity for the purposes of correlation. It also lets you decide how much time should pass before inactive issues are closed.
```

Comment:
"The incident intelligence general settings page"? Should any of this be bold or initial cap per style guide?

Reply:
Also "purposes of correlation" I would think there would be more than one purpose besides correlation? Or just "purpose"?

Reply:
I don't think it should be bolded. Also, we don't initial cap "incident intelligence" anymore.

Reply:
oh, i thought "settings" was the actual name of the UI page. Never mind!